common/core/src/errors.ts

```diff
@@ -430,3 +430,16 @@ export class OperationCancelledError extends Error {
   }
 }
 
+/**
+ * Error thrown when a DPS registration operation fails
+ *
+ * @augments {Error}
+ */
+export class DpsRegistrationFailedError extends Error {
```

Comment:
>DpsRegistrationFailedError [](start = 13, length = 26)

We shouldn't be using the DPS acronym. DeviceProvisioningError? DeviceRegistrationFailedError?